src/MR.AspNet.Identity.EntityFramework6.Model/MR.AspNet.Identity.EntityFramework6.Model.csproj

```diff
@@ -1,16 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
+  <Import Project="../../build/version.props" />
```

Comment:
"common.props" please! It includes "version.props" and adds stuff like "PackageProjectUrl" and the like.